java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -267,4 +273,100 @@ public void info_with_empty_InfoOptions_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(testPayload, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void sadd_returns_success() {
+        // setup
+        String key = "testKey";
+        String[] members = new String[] {"testMember1", "testMember2"};
+        String[] arguments = ArrayUtils.addFirst(members, key);
+        Long value = 2L;
+
+        CompletableFuture testResponse = mock(CompletableFuture.class);
+        when(testResponse.get()).thenReturn(value);
+
+        // match on protobuf request
+        when(commandManager.<String>submitNewCommand(eq(SAdd), eq(arguments), any()))
```

Comment:
`<String>` should probably be `<Long>`. This would allow you to give a proper type to the testResponse variable too (`CompletableFuture<Long>` instead of `CompletableFuture`).